bin/storm.py

```diff
@@ -754,18 +827,24 @@ def parse_config_opts(args):
     elif token == "--config":
       global CONFFILE
       CONFFILE = curr.pop()
+    elif token == "--jars":
```

Comment:
It would be better to add its usage to comment doc of `jar` and `sql` since print_usage(jar) will print out the usage of jar, and so on sql.